apps/dsp-app/src/app/workspace/resource/values/third-party-iiif/third-party-iiif.component.ts

```diff
@@ -0,0 +1,79 @@
+import { ChangeDetectorRef, Component, EventEmitter, Input, OnDestroy, OnInit, Output } from '@angular/core';
+import { FormBuilder, FormControl, Validators } from '@angular/forms';
+import { Subject } from 'rxjs';
+import { takeUntil } from 'rxjs/operators';
+import { iiifUrlValidator, infoJsonUrlValidatorAsync, previewImageUrlValidatorAsync } from './iiif-url-validator';
+import { IIIFUrl } from './third-party-iiif';
+
+@Component({
+  selector: 'app-third-part-iiif',
+  templateUrl: './third-party-iiif.component.html',
+  styleUrls: ['./third-party-iiif.component.scss'],
+})
+export class ThirdPartyIiifComponent implements OnInit, OnDestroy {
+  @Input() iiifUrlValue = '';
+  @Output() afterControlInit = new EventEmitter<FormControl<string>>();
+
+  iiifUrlControl: FormControl<string>;
+
+  previewImageUrl: string | null;
+  formStatus: 'VALIDATING' | 'LOADING' | 'IDLE' = 'IDLE';
+
+  private _destroy$ = new Subject<void>();
+
+  readonly validatorErrors = [
+    { errorKey: 'required', message: 'This field is required' },
```

Comment:
this error is already managed by HumanReadableError, you can remove it

Reply:
Nice, tx.